clubs_guides/urls.py

```diff
@@ -1,3 +1,29 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+# Root URL: `/api/clubsguides` (this can be changed in the teach/urls.py file)
+# NOTE: The patterns below are tacked onto the root url mentioned above
+
+urlpatterns = [
+    # `/` - Gets a list of clubs guides
+    url("^$", ClubsGuidesListView.as_view(), name="clubsguide-list"),
+
+    # `/:pk` - Gets a single clubs guide by its primary key (the named group is
+    # called pk by default in Django) viz. `id`
+    url(r"^(?P<pk>[0-9]+)", ClubsGuideView.as_view(), name="clubsguide"),
```

Comment:
why are we using `pk` here, rather than `id`? Especially if the route docs talk about `clubguides/:id`, and if the databases uses `id`, having the URL matcher use `pk` seems a bit odd.